common/djangoapps/student/management/commands/export_staff_users.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import absolute_import, print_function
+
+import logging
+from datetime import datetime, timedelta
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.core.mail.message import EmailMultiAlternatives
+from django.template.loader import get_template
+from os import remove
+
+from djqscsv import write_csv
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from student.models import CourseAccessRole
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+
+    help = """
+    This command will export a csv of all users who have logged in within the given days and
+    have staff access role in active courses (Courses with end date in the future).
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('days', type=int, help='Indicate the login time period in days starting from today')
```

Comment:
Please define 7 as  `default` in the `add_argument` method.
```
parser.add_argument("-v", "--verbosity", action="count", default=0,
                    help="increase output verbosity")
```
https://docs.python.org/2/howto/argparse.html